lib/ansible/modules/windows/win_msi.py

```diff
@@ -61,6 +60,9 @@
             - true
             - false
         default: false
+notes:
+- Check-mode support is currently not supported.
+- Please look into C(win_package) instead, this package will be deprecated in Ansible v2.3.
```

Comment:
`M(win_package)` for Module links.

Please update deprecation section of `CHANGELOG.md` for 2.3.